source/module_io/read_input_item_md.cpp

```diff
@@ -107,22 +108,92 @@ void ReadInput::item_md()
         read_sync_bool(mdp.md_restart);
         this->add_item(item);
     }
+    {
+        Input_Item item("lj_rule");
+        item.annotation = "combination rules used to construct the parameter matrix for LJ potential";
+        item.check_value = [](const Input_Item& item, const Parameter& para) {
+            if (para.input.esolver_type == "lj" && para.input.mdp.lj_rule != 1 && para.input.mdp.lj_rule != 2)
+            {
+                ModuleBase::WARNING_QUIT("ReadInput", "lj_rule must be 1 or 2");
+            }
+        };
+        read_sync_int(mdp.lj_rule);
+        this->add_item(item);
+    }
     {
         Input_Item item("lj_rcut");
         item.annotation = "cutoff radius of LJ potential";
-        read_sync_double(mdp.lj_rcut);
+        item.read_value = [](const Input_Item& item, Parameter& para) {
+            size_t count = item.get_size();
+            for (int i = 0; i < count; i++)
+            {
+                para.input.mdp.lj_rcut.push_back(std::stod(item.str_values[i]));
```

Comment:
You can avoid using push_back by first resize then std::transform, in general, but I think it will not be bottleneck of performance here.